android/filamat-android/CMakeLists.txt

```diff
@@ -2,9 +2,14 @@ cmake_minimum_required(VERSION 3.4.1)
 
 set(FILAMENT_DIR ${FILAMENT_DIST_DIR})
 
-add_library(filamat STATIC IMPORTED)
-set_target_properties(filamat PROPERTIES IMPORTED_LOCATION
-        ${FILAMENT_DIR}/lib/${ANDROID_ABI}/libfilamat.a)
+set(FILAMAT_VERSION "filamat")
```

Comment:
It's not really a version, it's more like a flavour...maybe simply call this `FILAMAT_NAME`?  Not a big deal.

Reply:
Good suggestion!